Fire updateFixed event on both fix and unfix

lingo opened the pull request:
I've updated the code to behave as I understood it would from the documentation.
Maybe this is useful to others?

Thanks!

lingo committed:
Fire updateFixed event on both fix and unfix

ShiraNai7 commented:
Seems reasonable, but `this.updateFixed()` shouldn't be called after `this.unfix()`, because unfixing resets the CSS styles and `this.updateFixed()` is used to update them. This could lead to an unfixed element with styles that only a fixed element should have.

I already have multiple DOM events planned for the next version:

- before fixing
- after fixing
- before unfixing
- after unfixing

Would that fulfil the requirements of your use case?

lingo committed:
Bugfix: don't call this.updateFixed()

"this.updateFixed() shouldn't be called after this.unfix()"
https://github.com/ShiraNai7/jquery-scrollfix/pull/2

ShiraNai7 commented:
DOM events have been implemented in 9ddad5f4. There is an example in the README.

Thanks for inspiring the idea.

lingo commented:
Looks great, thanks a lot! I'll make sure my code is using these.
